forest/cli/src/cli/attach_cmd.rs

```diff
@@ -222,12 +223,12 @@ async fn send_message(
 ) -> Result<MpoolPushMessageResult, jsonrpc_v2::Error> {
     let (from, to, value) = params;
 
-    let parsed_string = FILAmount::from_str(&value).map_err(|err| anyhow::anyhow!("{err:?}"))?;
+    let value = humantoken::parse(&value)?;
 
     let message = Message_v3 {
         from: Address::from_str(&from)?.into(),
         to: Address::from_str(&to)?.into(),
-        value: TokenAmount::from(parsed_string.value).into(),
+        value: value.into(), // Convert between TokenAmount2 and TokenAmount3
```

Comment:
This is converting from `forest_shim::econ::TokenAmount` to `fvm_shared3::econ::TokenAmount`.